docs/configuration/http_filters/router_filter.rst

```diff
@@ -219,6 +219,8 @@ prefix <config_http_conn_man_stat_prefix>` comes from the owning HTTP connection
   no_cluster, Counter, Total requests in which the target cluster did not exist and resulted in a 404
   rq_redirect, Counter, Total requests that resulted in a redirect response
   rq_total, Counter, Total routed requests
+  flow_control_paused_downstream_reads_total, Counter, Total times requests backed up enough to pause reading from downstream.
```

Comment:
We had cluster counters for when we stopped reading from upstream but not when upstream stopped reads from downstream.

Adding them now.  I think does mean we'll have somewhat redundant TCP and HTTP stats where the cluster level "downstream backing up" stats match the "upstream stopped reading" stats and vice versa but it'll make the H2 case easier to understand for sure.